src/sharing/components/Recipient.jsx

```diff
@@ -27,12 +33,17 @@ export const RecipientsAvatars = ({ recipients, link, size, className }) => {
             .slice(MAX_DISPLAYED_RECIPIENTS)
             .map(recipient => getDisplayName(recipient))}
           size={size}
+          onClick={onClick}
```

Comment:
Really don't like having to pass down this props everywhere

Reply:
The `onClick` no longer goes down to the avatars, it stays one level above. It's still a bit of drilling, but I'm not sure it makes sense to keep it anywhere above that?